src/shared/components/GeoPlot.tsx

```diff
@@ -0,0 +1,54 @@
+// Libraries
+import {FunctionComponent} from 'react'
+import {Config, Table} from '@influxdata/giraffe'
+
+// Types
+import {GeoViewProperties} from 'src/types'
+
+interface Props {
+  children: (config: Config) => JSX.Element
+  fluxGroupKeyUnion?: string[]
+  table: Table
+  viewProperties: GeoViewProperties
+}
+
+const GeoPlot: FunctionComponent<Props> = ({
+  children,
+  table,
+  viewProperties,
+}) => {
+  const {
+    layers,
+    zoom,
+    allowPanAndZoom,
+    detectCoordinateFields,
+    mapStyle,
+  } = viewProperties
+  const tileServerConfiguration = {
+    tileServerUrl: 'mapboxurlhere',
```

Comment:
maybe just leave this an empty string 🤷‍♂️

Reply:
Updated it :)